webkit_server.py

```diff
@@ -513,12 +520,47 @@ def issue_command(self, cmd, *args):
     self._writeline(cmd)
     self._writeline(str(len(args)))
     for arg in args:
-      arg = str(arg)
-      self._writeline(str(len(arg)))
-      self._sock.sendall(arg.encode("utf-8"))
+        self.send_arg(arg)
 
     return self._read_response()
 
+  def send_arg(self, arg):
+    """ Send each arg for args in issue_command """
+    arg = self.prepare_for_socket_sendall(arg)
+    self._writeline(str(len(arg)))
+    self._sock.sendall(arg)
+
+  def prepare_for_socket_sendall(self, arg):
+    """ Deal with the unicode and bytes problem in Python 2 and Python 3 for socket.sendall """
+
+    if PY2:
+      if type(arg) != unicode:
```

Comment:
While this would be a problem in other languages such as C++, where identifiers need to be defined at compilation time even if they are never used at runtime, this should not be an issue for Python, since here definitions of identifiers are only checked when encountered, and not during the syntax check.

In Python, it is also common to import packages only when they are required, and therefore `import` statements can be nested into `if` statements, indicating that an identifier does not have to be defined until it is encountered by the interpreter.

Therefore, I do not expect to find any problems with this line of code. In addition, I have successfully tested the suggested changes while observing no error messages or warnings.